content/docs/tutorials/smartmodule-development.md

```diff
@@ -176,7 +199,30 @@ Consuming records from the beginning of topic 'filter-test'
 42
 ```
 
-You now know  the development workflow for SmartModules with `smdk`. You can now generate your own project and process your own data.
+## Publish to SmartModule Hub
+
+As bonus, let's [publish] this SmartModule to [SmartMoudle Hub].
```

Comment:
I've got 2 pieces of feedback here:

1) Typo in the link text `SmartMoudle` -> `SmartModule`.

2) I like the idea of having a walkthrough for publishing, but I don't think here is the place to do that. This tutorial introduces the concept of customizing SmartModules to new (beginner) users of SmartModules, so including publish isn't very useful in this context. We should have a separate tutorial about publishing, that includes guidelines for producing quality, usable SmartModules.

Experienced readers are aware when they read content for beginners. But it isn't reasonable to expect beginners to know what is important when learning new concepts, if different contexts are combined. I think we definitely want this tutorial to be only beginning steps.

I also don't think we want dozens of regex-filter SmartModules published because people are following the tutorial closely.

Reply:
I'll remove. The fact that we don't have the ability to remove SmartModules from the Hub is going to generate a bunch of needless modules in the Hub that nobody cares for.